index.html

```diff
@@ -237,10 +238,18 @@ <h3><span></span>Skills</h3>
               </span>
             </div>
             <div>
-              <form action="">
-                <input type="text" id="fname" name="fname" placeholder="Full name">
-                <input type="text" id="email" name="email" placeholder="Email address">
-                <textarea name="" id="" cols="10" rows="5" placeholder="Enter text here"></textarea>
+              <form action="https://formspree.io/f/xqkjbgbd"  method="post">
+                <ul>
+                  <li>
+                    <input type="text" id="fname" name="fname" placeholder="Full name" maxlength="30" required>
+                  </li>
+                  <li>
+                    <input type="email" id="email" name="email" placeholder="Email address" required>
+                  </li>
+                  <li>
+                    <textarea name="textarea" id="textarea" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
- [ ] I would suggest you please, implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.
### Figma template
![gihms](https://user-images.githubusercontent.com/81727988/194015291-ae7da499-b0ab-4c88-abef-1c1d3ec95082.png)